src/Parser/JsonMap.php

```diff
@@ -50,7 +51,7 @@ public function __construct(Config $config, LoggerInterface $logger, ?ParserInte
                 ));
             }
 
-            $mappers[$type] = new Mapper($mapping, $type);
+            $mappers[$type] = new Mapper($mapping, true, $type);
```

Comment:
v CsvTable kniznici pribudol parameter `writerHeader`, ... predtym sa hlavicka zapisovala vzdy, teraz je to volitelne, takze davame `true` -> nic sa nemeni.